pkg/args/args.go

```diff
@@ -17,11 +19,29 @@ func ReadArgsAndValidate() {
 		return
 	}
 
+	//check if the directory already exists
 	isFileExists,err:=checkDirectoryExists(fileName)
 	if(isFileExists){	
-		fmt.Println("file already exists")
+		//configure prompt
```

Comment:
Can the prompt be moved to another function?

Reply:
@AliAkberAakash Have moved the prompt logic to a new function in the latest commit